src/main/java/org/spongepowered/common/mixin/core/item/inventory/MixinItemStack.java

```diff
@@ -49,9 +61,13 @@
     @Shadow public int stackSize;
 
     @Shadow public abstract int getItemDamage();
+
```

Comment:
That is the sponge formatter doing that I believe.
________________________________
From: Gabriel Harris-Rouquette<mailto:notifications@github.com>
Sent: ‎5/‎18/‎2015 5:06 PM
To: SpongePowered/SpongeCommon<mailto:SpongeCommon@noreply.github.com>
Cc: Steven Downer<mailto:grinch@outlook.com>
Subject: Re: [SpongeCommon] [WIP] AuthorData implementation and fixes for PagedData (#38)

> @@ -49,9 +61,13 @@
>      @Shadow public int stackSize;
>
>      @Shadow public abstract int getItemDamage();
> +

don't add spaces here please.

---
Reply to this email directly or view it on GitHub:
https://github.com/SpongePowered/SpongeCommon/pull/38/files#r30552358